### PR TITLE
New network Core

### PR DESCRIPTION
**Pull request by Tguntenaar**
# Description

# Issues

Fixes #
Refs #

# Checklist:

- [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

**Review by julien51 — COMMENTED** [review]
Thanks @Tguntenaar !
I think there are still a few important parts that are missing. 

Please run ` yarn run check` in this folder to check what you might be missing
Also, please make sure you verify all contracts!

File: packages/networks/src/networks/core.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+      base: 'https://scan.coredao.org/',
+      token: (address, holder) =>
+        `https://scan.coredao.org/token/${address}?a=${holder}`,
+      transaction: (hash) => `https://scan.coredao.org/tx/${hash}`,
+    },
+  },
+  featured: false,
+  id: 1116,
+  isTestNetwork: false,
+  multisig: '0x0eCbE42d1B8c9e56a9d358D3Ac1b5aaFAf5f07d5',
```
Comment (clemsos) [inline packages/networks/src/networks/core.ts:20]: I see that multisig only have a single signer. The best will be to create the safe using the available script `yarn hardhat safe:create` or if not possible add the signers specified here https://github.com/unlock-protocol/unlock/blob/2a1715bbcb325965c8b69ea9473303ae034081f7/governance/helpers/multisig.js#L22-L32

Comment (clemsos) [reply packages/networks/src/networks/core.ts:20]: Also there is a `yarn hardhat safe:check` to make sure everything is in order

File: packages/networks/src/networks/core.ts [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    name: 'CORE',
+    symbol: 'CORE',
+    wrapped: '0x40375C92d9FAf44d2f9db9Bd9ba41a3317a2404f',
+  },
+  provider: 'https://rpc.coredao.org/',
+  publicLockVersionToDeploy: 14,
+  publicProvider: 'https://rpc.coredao.org/',
+  startBlock: undefined,
+  subgraph: {
+    endpoint: 'https://thegraph.coredao.org',
```
Comment (clemsos) [inline packages/networks/src/networks/core.ts:34]: the endpoint should be for the actual graph that has been deployed to track Unlock contracts events on the network

**Review by clemsos — COMMENTED** [review]
Hey, LG so far ! You can see what is missing by running `yarn hardhat unlock:info` from the `governance` folder.

**Comment by clemsos** [comment]
hello @Tguntenaar, any update here?

**Comment by Tguntenaar** [comment]
Hey @clemsos ! We weren't able to figure it out with the current state of the docs and limited development resources. Our team has gone with a different solution for now. Hope to see Core DAO added in the future.

**Comment by clemsos** [comment]
> Hey @clemsos ! We weren't able to figure it out with the current state of the docs and limited development resources. Our team has gone with a different solution for now. Hope to see Core DAO added in the future. 

Sorry to hear ! We'll work on the docs to make it simpler

**Pull request closed without merging**